sdk/playwrighttesting/microsoft-playwright-testing/src/reporter/mptReporter.ts

```diff
@@ -380,6 +381,7 @@ class MPTReporter implements Reporter {
       this.envVariables.userId = mptTokenDetails.oid;
       this.envVariables.userName = mptTokenDetails.userName;
       this.envVariables.region = ReporterUtils.getRegionFromAccountID(this.envVariables.accountId!);
+      validateMptPAT();
```

Comment:
Can be moved above, also this call will exit the reporter process. Ideally, we want to terminate the reporter execution and let test execution continue.